### PR TITLE
[9.0] [MIG]  [mrp_bom_product_details]

### PR DESCRIPTION
**Pull request by misern2**
Bom product details
===================

This module adds product price and stock to bom view.

Usage
=====

* Go to Manufacturing > Products > Bill of Materials

### CODE REVIEW

File: mrp_bom_product_details/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#########################################################################
```
Comment (LoisRForgeFlow) [inline mrp_bom_product_details/models/__init__.py]: Can you change to short headers? Follow the <a href="https://github.com/OCA/maintainer-tools/tree/master/template/module">template</a>.

File: mrp_bom_product_details/README.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Installation
+============
+
+To install this module, you just need to select the module and insure yourself dependancies are available.
+
+Configuration
+=============
+
+No particular configuration to use this module.
```
Comment (LoisRForgeFlow) [inline mrp_bom_product_details/README.rst]: Remove this section since it's not needed.

File: mrp_bom_product_details/README.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Contributors
+------------
+
+* Solutions Libergia inc.  <david.dufresne@savoirfairelinux.com>
+* QubiQ  <marsal.isern@qubiq.es>
+
+
+Icon
+----
+* http://en.wikipedia.org/wiki/File:People%27s_Action_Party_of_Singapore_logo.svg
```
Comment (LoisRForgeFlow) [inline mrp_bom_product_details/README.rst]: Use the OCA logo following the template.

File: mrp_bom_product_details/views/mrp_bom_product_details.xml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
```
Comment (LoisRForgeFlow) [inline mrp_bom_product_details/views/mrp_bom_product_details.xml]: data tag can be removed

File: mrp_bom_product_details/views/mrp_bom_product_details.xml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    
+		<record id="mrp_bom_tree_view" model="ir.ui.view">
+			<field name="name">mrp.bom.tree</field>
```
Comment (LoisRForgeFlow) [inline mrp_bom_product_details/views/mrp_bom_product_details.xml]: Try to indent with four spaces.

**Review by LoisRForgeFlow — COMMENTED** [review]
Thank you for the contribution! Some comments:

**Comment by misern2** [comment]
Thank you for your comments @lreficent . It's been very useful. 👌

File: mrp_bom_product_details/README.rst [hunk 6]
```diff
@@ -0,0 +1,46 @@
+Bom product details
```
Comment (cubells) [inline mrp_bom_product_details/README.rst:6]: Follow this file https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
please

File: mrp_bom_product_details/README.rst [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+===================
+
+This module adds product price and stock to bom view.
+
+Installation
+============
+
+To install this module, you just need to select the module and insure yourself dependancies are available.
```
Comment (cubells) [inline mrp_bom_product_details/README.rst]: Line too long

File: mrp_bom_product_details/__init__.py [hunk 8]
```diff
@@ -1,23 +1,5 @@
...
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU General Public License for more details.
-#
-#    You should have received a copy of the GNU General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
-import mrp_bom_product_details
+from . import models
```
Comment (cubells) [inline mrp_bom_product_details/__init__.py]: Newline is needed

File: mrp_bom_product_details/__openerp__.py [hunk 9]
```diff
@@ -21,16 +21,18 @@
...
     'name': 'Bom product details',
-    'version': '0.1',
-    'author': "Solutions Libergia inc.,Odoo Community Association (OCA)",
+    'version': '9.0.1.0.0',
+    'author':  "Solutions Libergia inc., "
+               "QubiQ, "
+               "Odoo Community Association (OCA),QubiQ",
     'license': 'GPL-3 or any later version',
     'category': 'Manufacturing',
     'description': """
```
Comment (cubells) [inline mrp_bom_product_details/__openerp__.py]: This key is not needed

File: mrp_bom_product_details/__openerp__.py [hunk 10]
```diff
@@ -21,16 +21,18 @@
...
     This module adds product price and stock to bom view
     """,
     'depends': ["base", "mrp"],
     'demo': [],
```
Comment (cubells) [inline mrp_bom_product_details/__openerp__.py]: remove empty keys

**Review by cubells — CHANGES_REQUESTED** [review]
Fast code review

**Comment by LoisRForgeFlow** [comment]
@misern2 could you check travis?

**Comment by misern2** [comment]
@lreficent I updated my local branch and the Travis error was solved by commit  5f51684 

I'm not sure if it's the best way to proceed. Maybe a cherry peek with that commit it would be better.

**Comment by LoisRForgeFlow** [comment]
@misern2 I see that you have added many commits related to other modules that shouldn't be here. Could you try to rebase against an updated OCA v9 branch? The the commit you are referring is already merged so it should be added with a proper rebase of your PR.

**Comment by misern2** [comment]
@lreficent  I made a rebase. The Travis build finally passed.

File: mrp_bom_product_details/__openerp__.py [hunk 11]
```diff
@@ -1,36 +1,20 @@
...
-    'license': 'GPL-3 or any later version',
-    'category': 'Manufacturing',
-    'description': """
+    'summary': """
     This module adds product price and stock to bom view
     """,
+    'version': '9.0.1.0.0',
+    'author':  "Solutions Libergia inc., "
+               "QubiQ, "
+               "Odoo Community Association (OCA),QubiQ",
```
Comment (LoisRForgeFlow) [inline mrp_bom_product_details/__openerp__.py]: QubiQ is duplicated.

File: mrp_bom_product_details/__openerp__.py [hunk 12]
```diff
@@ -1,36 +1,20 @@
...
+    'license': 'GPL-3 or any later version',
```
Comment (LoisRForgeFlow) [inline mrp_bom_product_details/__openerp__.py]: Be consistent with headers: `"AGPL-3"`

**Comment by LoisRForgeFlow** [comment]
@misern2 Great, this is ready to review now! Thanks!

**Review by LoisRForgeFlow — COMMENTED** [review]
Some comments

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot, LGTM now :+1: 

Thanks!

**Comment by LoisRForgeFlow** [comment]
@cubells Could you update your review, please?

File: mrp_bom_product_details/README.rst [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+
+This module adds product price and stock to bom view.
+
+
+Installation
+============
+
+To install this module, you just need to select the module and insure
+yourself dependancies are available.
```
Comment (cubells) [inline mrp_bom_product_details/README.rst]: s/dependancies/dependencies/g

File: mrp_bom_product_details/README.rst [hunk 14]
```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```
Comment (cubells) [inline mrp_bom_product_details/README.rst]: Change this line by

`:target: https://www.gnu.org/licenses/agpl`

as is indicated in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: mrp_bom_product_details/__init__.py [hunk 15]
```diff
@@ -1,23 +1,5 @@
 # -*- coding: utf-8 -*-
+# © 2015 - Solutions Libergia inc.
```
Comment (cubells) [inline mrp_bom_product_details/__init__.py]: It's recommended to replace symbol © by `Copyright`

File: mrp_bom_product_details/models/__init__.py [hunk 16]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 - Solutions Libergia inc.
```
Comment (cubells) [inline mrp_bom_product_details/models/__init__.py]: It's recommended to replace symbol © by `Copyright`

File: mrp_bom_product_details/models/mrp_bom_product_details.py [hunk 17]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2015 Oihane Crucelaegui - AvanzOSC
```
Comment (cubells) [inline mrp_bom_product_details/models/mrp_bom_product_details.py]: It's recommended to replace symbol © by `Copyright`

**Review by cubells — CHANGES_REQUESTED** [review]
Minor changes. 

Ready to approve.

**Comment by misern2** [comment]
@cubells Can you update your review, please ?

Thanks!.

**Review by cubells — APPROVED** [review]
👍 

LGTM

Code review and test

**Pull request merged**